README.md

```diff
@@ -62,6 +62,41 @@ python python/server_vllm.py
 ```
 Fast suggestions are optional; you can use `python/server.py` to run `llmstep` without vLLM.
 
+## Local inference with llama.cpp
+The standard server and vLLM server are intended to run on GPUs. However, `llmstep` also includes `python/server_ggml.py`, an experimental [llama.cpp](https://github.com/ggerganov/llama.cpp) based server intended to run efficiently on CPUs. Currently, the llama.cpp server supports only one tactic suggestion per `llmstep` call. 
+
+Configuring `python/server_ggml.py` requires a few extra steps. Note that the official [llmstep model](https://huggingface.co/wellecks/llmstep-mathlib4-pythia2.8b) is not compatible with the llama.cpp server. Instead, download the experimental [zhangirazerbayev/open-llama-3b_next-tactic_dev0.2](https://huggingface.co/zhangirazerbayev/open-llama-3b_next-tactic_dev0.2) as follows:
+```bash
+# Make sure you have git-lfs installed (https://git-lfs.com)
+git lfs install
+git clone https://huggingface.co/zhangirazerbayev/open-llama-3b_next-tactic_dev0.2
+```
+The following steps will install llama.cpp and convert the above model to an executable binary. 
+```bash
+git clone https://github.com/ggerganov/llama.cpp
+cd llama.cpp
+make
+
+# install llama.cpp python dependencies
+python3 -m pip install -r requirements.txt
+
+# convert the model you downloaded to ggml FP16 format
+python3 convert.py $PATH_TO_MODEL
+```
+This will produce a file named `ggml-model-f32.bin` in the `$PATH_TO_MODEL` directory. 
+
+If the above steps do not work for you, please, consult the more detailed instructions in the [llama.cpp](https://github.com/ggerganov/llama.cpp) repository.
+
+In the following step, you will quantize your model to a reduced precision format. The available formats are `F16, Q8_0, Q5_1, Q5_0, Q4_1, Q4_0`, with lower precision formats trading off accuracy for latency and memory. I would recommend starting with `Q4_0`, and increasing precision if your hardware handles lower precisions comfortably. 
+```bash
+./quantize $PATH_TO_MODEL/ggml-model-f32.bin $PATH_TO_QUANTIZED
```

Comment:
This is missing the `type` argument for quantize (e.g. `q4_0`) after the $PATH_TO_QUANTIZED.